code/AnalyticsDataframe.py

```diff
@@ -125,17 +117,56 @@ def is_name_matched(name: list, mean: dict, var: dict):
 # 3  NaN  NaN  NaN
 # 4  NaN  NaN  NaN
 
+##
+## Test case 1 - validate statistical properties of generated predictor matrix
+##
+ad2 = AnalyticsDataframe(10000, 3, ["xx1", "xx2", "xx3"], "yy")
 C = np.array([[1, -0.5, 0.3],
               [-0.5, 1, 0.2],
               [0.3, 0.2, 1]])
 ad2.update_predictor_normal(predictor_name_list=["xx1", "xx2", "xx3"],
-                            mean_dict={"xx1": 1, "xx2": 2, "xx3": 5},
-                            std_dict={"xx1": 0.5, "xx2": 1, "xx3": 1},
-                            correlation_matrix=C)
-print(ad2.predictor_matrix)
-#         xx1       xx2       xx3
-# 0  1.865383  2.218858  6.750145
-# 1  0.562747  1.535777  6.324234
-# 2  1.381225  1.521372  7.069359
-# 3  0.937004  1.359714  5.233999
-# 4  0.853955  1.408445  5.944211
+                            mean=[1, 2, 5],
+                            covariance_matrix=C)
+test_matrix_1 = ad2.predictor_matrix
+print(test_matrix_1.mean())
+# xx1    1.003769
+# xx2    2.007161
+# xx3    5.008807
+
+print(test_matrix_1.corr())
+#           xx1       xx2       xx3
+# xx1  1.000000 -0.501385  0.300073
+# xx2 -0.501385  1.000000  0.203265
+# xx3  0.300073  0.203265  1.000000
+
+##
+## Test case 2 - Test handling of variable lists
+##
+
+ad3 = AnalyticsDataframe(100, 5)
```

Comment:
Instead of the shape of `(100, 5)`, a shape of `(100, 3)` should be used in order to control variables that cause an error, that is, to highlight the **sole** cause of an error.

Reply:
Thanks for pointing it out! Will fix.